docs/dataset/advanced-usage/upload-embeddings.md

```diff
@@ -0,0 +1,65 @@
+---
+icon: kolena/classification-16
+---
+
+# :kolena-classification-16: Uploading Custom Embeddings
+
+This guide explains how to upload your own embeddings to Kolena using the Kolena SDK.
+Please ensure you have the SDK installed.
+[Instructions for installing the SDK are available here.](https://docs.kolena.com/installing-kolena/)
+
+## Step 1: Import the Embedding Upload Function
+
+To upload embeddings, use the `upload_dataset_embeddings` function from Kolena. You can import
+it with the following code:
+```
+from kolena._experimental.search import upload_dataset_embeddings
+```
+
+## Step 2: Prepare the Required DataFrame
+
+The DataFrame you upload should have two columns:
```

Comment:
there could be multiple id fields (say `locator` + `person_id`)

Reply:
Removed two columns. Added a line to specify it can be multiple id fields.